src/kv/view_containers.h

```diff
@@ -59,20 +73,49 @@ namespace kv
         ok = false;
         break;
       }
+
+      for (const auto& [map_name, map_ptr] : new_maps)
```

Comment:
Minor: It feels like this should be done before we look at the conflicts (`prepare()`) between keys within maps?

Reply:
I think we can do these in either order, I don't see a reason to prefer doing one first? Though looking at it in the diff I've just noticed we're doing this in the loop, once-per-view, which is wrong.